src/java/io/bazel/rulesscala/jar/BUILD

```diff
@@ -0,0 +1,5 @@
+java_binary(name = "jar",
```

Comment:
That is good to know about instead of using system `zip`, but what about making manifests? Seems nice to reuse that bit?